topics/imaging/tutorials/astronomy-source-extractor/tutorial.md

```diff
@@ -0,0 +1,226 @@
+err_option = 'none'             # Use 'thresh' as an absolute threshold
+```
+
+## Data Requirements 
+
+The source-extractor tool operates on a single image file, typically for astronomy, a sky image containing luminous sources. Optionally, users can supply a mask and/or a filter. 
+
+**Image:** 
+- Preferrably: light sources on a dark background.
+- Format: a single-channel 2D array stored as ```.tiff``` or ```.fits```. 
+
+**Mask (Optional):** 
+- Masks regions affected by bright sources (e.g. stars) to improve background estimation. 
+- Pixels with
+``` python
+value > maskthresh
+```
+or boolean ```True``` are masked.
+- Format: a single-channel 2D array stored as ```.tiff``` or ```.fits```.  
+
+**Filter Kernel (Optional):** 
+The filter kernel is used to smooth the input image, which can enhance the detection of faint and extended sources. However, in crowded fields, filtering may reduce performance by blending nearby objects.
+
+- If ```Filter Case``` is set to ```none```, no filtering is applied.
+- If ```Filter Case``` is ```default```, a built-in smoothing kernel is used:
+```markdown
+1 2 1
+2 4 2
+1 2 1
+```
+- If ```Filter Case``` is ```file```, you must provide a custom 2D array stored as plain text ```.txt``` file. The file should contain whitespace-separated values and must be readable with:
+```python
+import numpy as np
+kernel = np.loadtxt("filter.txt")
+```
+
+
+> <comment-title> Checking the metadata of an image </comment-title>
+> Tip 1: Use {% tool [Show image info](toolshed.g2.bx.psu.edu/repos/imgteam/image_info/ip_imageinfo/5.7.1+galaxy1) %} to inspect ```.tiff``` metadata. Required:
+>
+> ``` RGB = false (1) ```
+> ``` Interleaved = false ```
+> ``` SizeZ = 1 ```
+> ``` SizeT = 1 ```
+> ``` SizeC = 1 ```
+>
+> Tip 2: Use {% tool [astropy fitsinfo](toolshed.g2.bx.psu.edu/repos/astroteam/astropy_fitsinfo/astropy_fitsinfo/0.2.0+galaxy2) %} to check ```.fits``` metadata. Required:
+> ```Dimensions (N, M) ```, where ```N``` and ```M``` are pixel dimensions in 2D. 
+{: .comment}
+
+## Getting data from DESI Legacy Surveys
+> <hands-on-title> Data Acquisition </hands-on-title>
+>
+> 1. Create a new history for this tutorial. You can rename the default unnamed history.
+> 
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+> 
+> 2. Run the {% tool [DESI Legacy Survey](toolshed.g2.bx.psu.edu/repos/astroteam/desi_legacy_survey_astro_tool/desi_legacy_survey_astro_tool/0.0.2+galaxy0) %} tool. 
+> 
+>    - **Important:** Choose the Data Product **Image**.
+> 
+>    The default values are used for this tutorial.
+>    The history now contains the ```.fits``` image file that is used as input for the source-extractor tool.
+{: .hands_on}
+
+## Running the Source-Extractor Tool
+
+Once you’ve selected the source-extractor tool, choose the input file named: ``` DESI Legacy Survey -> Image fits ```. After the tool has finished running, several output images and data products will be available:
```

Comment:
minor detail: this is the first time (I think) that the user is addressed as "you" instead of user. It would be good to be consistent in addressing the reader